confidential/86cb680/86cb680.bug

```diff
@@ -45,4 +45,4 @@ fault-codes:
   - PROGRAMMING:LOGIC
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:MONITORING
```

Comment:
Why monitoring rather than behaving incorrectly?

Reply:
> network failure would mean that the condition is reported as false

Because it makes the system believe (not the end-user in this case) in an incorrect status.

Reply:
Agreed on monitoring during the meeting